### PR TITLE
Add some more ESRI projection method mappings

### PR DESCRIPTION
**Pull request by nyalldawson**
Adds
- Mercator_Variant_A
- Mercator_Variant_C
- Transverse_Cylindrical_Equal_Area

### CODE REVIEW

File: scripts/build_esri_projection_mapping.py [hunk 1]
```diff
@@ -674,23 +632,67 @@
...
+        - Central_Meridian: EPSG_NAME_PARAMETER_LONGITUDE_OF_NATURAL_ORIGIN
+        - Scale_Factor: EPSG_NAME_PARAMETER_SCALE_FACTOR_AT_NATURAL_ORIGIN
+        - Latitude_Of_Origin: EPSG_NAME_PARAMETER_LATITUDE_OF_NATURAL_ORIGIN
 
-# Gnomonic_Auxiliary_Sphere: not handled
+# Missing/unclear mappings
+
+# Hammer_Aitoff: possibly hammer?
+# Hammer_Ellipsoidal: possibly hammer?
+# Eckert_Greifendorff: +proj=hammer +W=0.25 +m=1
```
Comment (rouault) [inline scripts/build_esri_projection_mapping.py]:
```suggestion
# Eckert_Greifendorff: +proj=hammer +W=0.25 +M=1
```

**Comment by rouault** [comment]
For the sake of being good open source citizens (and have an up-to-date reference), provided that https://github.com/Esri/projection-engine-db-doc/pull/17 is merged, we should probably contribute updates to https://github.com/Esri/projection-engine-db-doc/blob/master/other/proj4-esri.txt, especially in the "PE projections not supported by PROJ4 " to make it smaller :-)  Good candidates that we have already mapped are: Behrmann, Plate_Carree, Equidistant_Cylindrical_Ellipsoidal, Mercator_Auxiliary_Sphere

**Comment by nyalldawson** [comment]
There's quite a few "*" (only in libproject flag) which can be removed too...

**Comment by nyalldawson** [comment]
(besides, I'd find it kind of amusing if ESRI repos had me as a contributor 🙃)

**Comment by rouault** [comment]
> I'd find it kind of amusing of ESRI repos had me as a contributor

ahah, be reassured, you're not alone: https://github.com/Esri/lerc/commits?author=rouault :-)

**Pull request merged**